ttlib/page.tt

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
+<html lang="en" class="h-100">
+  <head>
+[% IF base -%]
+    <base href="[% base %]">
+[% END -%]
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="Mark Otto, Jacob Thornton, and Bootstrap contributors">
+    <meta name="generator" content="Hugo 0.84.0">
```

Comment:
Cargo culted? We're not using Hugo here.

Reply:
Maybe it's a security measure to distract vulnerability scanners?

Reply:
Yeah, I'll fix that. It's there because I copied the code from a Bootstrap example site.

I really wouldn't describe it as cargo-culting - I know exactly what all of those lines do 😊